backend/api/tests/test_models.py

```
@@ -0,0 +1,1415 @@
+        self.assertIsNotNone(anfrage.anfrage_datum)
+
+    def test_fall_one_to_one_enforced(self):
+        Anfrage.objects.create(
+            anfrage_weg="Mail",
+            anfrage_ort="LS",
+            anfrage_person="B",
+            anfrage_art="B",
+            fall=self.fall,
+        )
+
+        with self.assertRaises(Exception):
```

Ruff (0.14.10)

Do not assert blind exception: `Exception`

(B017)